src/library/dlgcoverartfullsize.h

```diff
@@ -14,8 +14,8 @@ class DlgCoverArtFullSize
           public Ui::DlgCoverArtFullSize {
     Q_OBJECT
   public:
-    DlgCoverArtFullSize(QWidget* parent = nullptr, BaseTrackPlayer* pPlayer = nullptr);
-    virtual ~DlgCoverArtFullSize();
+    explicit DlgCoverArtFullSize(QWidget* parent, BaseTrackPlayer* pPlayer = nullptr);
+    ~DlgCoverArtFullSize() override = default;
```

Comment:
This is redundant

Reply:
No

Reply:
Why not? Doesn't the compiler generate the default destructor in any case?